tests/vars.sample.yaml

```diff
@@ -25,7 +25,7 @@ oc_header: |
   eval $(crc oc-env)
 
 # Source MariaDB Galera cluster VIP for DB exports.
-source_mariadb_ip: 192.168.122.99 #CUSTOMIZE_THIS
+source_mariadb_ip: 172.17.0.100 #CUSTOMIZE_THIS
```

Comment:
Can we also edit it in the docs? https://github.com/openstack-k8s-operators/data-plane-adoption/blob/d1ea65f3c8c65a6422eb4b5d233430a2e16cccab/docs_user/modules/proc_migrating-databases-to-mariadb-instances.adoc#L46 

Otherwise LGTM

Reply:
https://github.com/openstack-k8s-operators/data-plane-adoption/pull/466 to address the issue